### PR TITLE
Add Z-Wave board announcement blog

### PR DESCRIPTION
**Pull request by googanhiem**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a blog post announcing the appointment of Paulus Schoutsen to the Z-Wave Alliance Board.
	- Highlighted the commitment to open standards and the development of Z-Wave connectivity hardware within the Home Assistant community.
	- Recognized Nabu Casa's achievement with the Innovation Enabler Award from the Thread Group.

### CODE REVIEW

File: source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Being part of this board allows us to play a bigger role in shaping Z-Wave's future. As a [Standards Development Organization (SDO)](https://z-wavealliance.org/z-wave-alliance-formalizes-standards-development-organization-and-announces-founding-members/) the Alliance is shaped by its members, and they brought us in for our connection to our community and expertise in open source. We will be giving a voice to Home Assistant users, open source development, and Open Home Foundation values.
+
+<!--more-->
+
+A tenth of households with Home Assistant are using Z-Wave today, accounting for an estimated 1.7 million Z-Wave devices. We've seen a growing interest in Z-Wave in the Home Assistant community.  This is why we have been driving forward the development of the open source [Z-Wave JS](https://github.com/zwave-js) stack, while also developing our own [Z-Wave connectivity hardware](https://www.home-assistant.io/blog/2024/05/08/zwave-is-not-dead/#range-testing-our-z-wave-stick-prototype).
```
Comment (frenck) [inline source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown]:
```suggestion
A tenth of households with Home Assistant are using Z-Wave today, accounting for an estimated 1.7 million Z-Wave devices. We've seen a growing interest in Z-Wave in the Home Assistant community. This is why we have been driving forward the development of the open source [Z-Wave JS](https://github.com/zwave-js) stack, while also developing our own [Z-Wave connectivity hardware](https://www.home-assistant.io/blog/2024/05/08/zwave-is-not-dead/#range-testing-our-z-wave-stick-prototype).
```

File: source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+As you might be well aware, we're very keen on open standards and are continuing to put a lot of work into our support of Zigbee, Matter, Thread, and Z-Wave. These standards are important for a number of reasons that align with our [Open Home mission](https://www.openhomefoundation.org/about/).
```
Comment (frenck) [inline source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown]:
```suggestion
As you might be well aware, we're very keen on open standards and are continuing to put a lot of work into supporting Zigbee, Matter, Thread, and Z-Wave. These standards are important for a number of reasons that align with our [Open Home mission](https://www.openhomefoundation.org/about/).
```

File: source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+These standards enable greater choice, giving our users access to decades of smart home devices to add to their homes. These standards are locally controlled and don't rely on the cloud, meaning when devices are connected through Home Assistant they provide privacy out-of-the-box.  Finally, there are a lot of these devices already in use in our homes, along with second-hand devices ready for a new home. Keeping a standard alive, keeps these devices alive and using what already exists is always the most sustainable option.
```
Comment (frenck) [inline source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown]:
```suggestion
These standards enable greater choice, giving our users access to decades of smart home devices to add to their homes. These standards are locally controlled and don't rely on the cloud, meaning when devices are connected through Home Assistant they provide privacy out-of-the-box. Finally, there are a lot of these devices already in use in our homes, along with second-hand devices ready for a new home. Keeping a standard alive, keeps these devices alive and using what already exists is always the most sustainable option.
```

File: source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+This is not the only example of our work being recognized by these standards organizations. Recently, the Thread Group selected Nabu Casa for an [Innovation Enabler Award](https://www.linkedin.com/posts/thread-group_threadgroup-internetofthings-smarthome-activity-7226619123235971072-8kHn?utm_source=share&utm_medium=member_desktop), which is only awarded to two companies a year, and recognizes the "creative potential of [our] ideas and the use of Thread technology".
```
Comment (frenck) [inline source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown]:
```suggestion
This is not the only example of our work being recognized by these standards organizations. Recently, the Thread Group selected Nabu Casa for an [Innovation Enabler Award](https://www.linkedin.com/posts/thread-group_threadgroup-internetofthings-smarthome-activity-7226619123235971072-8kHn), which is only awarded to two companies a year, and recognizes the "creative potential of [our] ideas and the use of Thread technology".
```

Comment (frenck) [reply source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown]: `[our]` seems to be linked to nothing?

File: source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+[We believe in Z-Wave](/blog/2024/05/08/zwave-is-not-dead/) and we will continually push it to align with the Open Home, so it can bring better choice, privacy, and sustainability to the home. If you want to learn more about the Z-Wave Alliance Board, read the appointment announcement here.
```
Comment (frenck) [inline source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown]: `read the appointment announcement here`

Was that intended to be a link?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown (hunk 4 above)
Comment (googanhiem) [reply source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown]: That's not meant to be markup, but yes looks similar. Just a text replacement in a quote (replacing "theirs" with "ours").

File: source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown (hunk 5 above)
Comment (googanhiem) [reply source/_posts/2024-09-24-we-are-joining-the-z-wave-alliance-board.markdown]: Still awaiting the link.

**Review by googanhiem — COMMENTED** [review]
Link won't be ready for a while, so I will add it later. 

Lets post this now.

**Review by frenck — APPROVED** [review]
Thanks, @googanhiem 👍 

../Frenck

**Pull request merged**